Sources/Swift/Transactions/SentryTraceOrigin.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+@objcMembers
+public class SentryTraceOrigin: NSObject {
```

Comment:
`h`: Again I would like to keep this internal, same as here: https://github.com/getsentry/sentry-cocoa/pull/4718#discussion_r1918222526

Reply:
Not possible because they are used by SentrySwiftUI